nixos/lib/test-driver/nixos_test_driver/nixos_test_driver.py

```diff
@@ -1,4 +1,10 @@
-#! /somewhere/python3
+"""
+NixOS test driver
+=================
+"""
+__version__ = "0.1"
```

Comment:
is there some way to give the driver a central version number that can be reused so we have only one definition? (e.g. nixos/lib/test-driver/nixos_test_driver/doc/conf.py has such a field, too)

Reply:
It could be re-imported from the module.